src/Menu/AdminMenuListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusBannerPlugin\Menu;
+
+use Sylius\Bundle\UiBundle\Menu\Event\MenuBuilderEvent;
+
+final class AdminMenuListener
+{
+    public function buildMenu(MenuBuilderEvent $event): void
+    {
+        $menu = $event->getMenu();
+
+        $newSubmenu = $menu
```

Comment:
I'm not sure if this is enough (what if for example someone has both plugins installed cms and banner ? ) Both have a submenu called "Section" (unless they are in a separate set of configurations like "Sales"/"Customer" etc)